react/wbooks/src/app/screens/Home/layout.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import { array } from 'prop-types';
+import cn from 'classnames';
+import { t } from 'i18next';
 
 import NavBar from '../../components/NavBar';
 import BookInfo from '../../components/BookInfo';
 
 import styles from './styles.module.scss';
 
 function Home({ books }) {
+  const areBooks = books.length > 0;
+
   return (
-    <div className={styles.container}>
+    <div className={cn({ 'background-wild-sand': areBooks, 'background-white': !areBooks })}>
       <NavBar />
-      <div className={`row ${styles.booksContainer}`}>
-        {books.map(({ author, title, id }) => (
-          <BookInfo author={author} title={title} className="m-right-7 m-top-7" key={id} />
-        ))}
-      </div>
+      {areBooks > 0 ? (
```

Comment:
this is bad, because areBooks is a boolean, and you're asking `true > 0` or `false > 0` in both cases the answer is false. it should be `areBooks` alone

Reply:
yes you are rigth! I forgot to adapt the new code I made